src/components/CampaignVisualisations/index.js

```diff
@@ -62,15 +62,11 @@ export default ({ visualisations }) => {
   );
 };
 
-export const CrowdFundingVisualistation = ({ startnextId, goal, ...props }) => {
-  const [crowdFunding] = useGetCrowdfundingDirectly(startnextId);
-
-  if (!crowdFunding) {
+export const CrowdFundingVisualistation = ({ project, goal, ...props }) => {
+  if (!project) {
```

Comment:
Maybe here, if there's no project key at all, we can query the startnext api? This way it can still support any other implementation where only the `startnextId` is passed? (if that still needs to be supported)

Reply:
exactly

Reply:
Done